### PR TITLE
Closes #104 Documentation: rendering and clarity issues in the reference page

### LINKED ISSUE #104: Documentation:  rendering and clarity issues in the reference page
### Please select a category the issue is focused on?

Function Documentation

### Let us know where something needs a refresh or put your idea here!

Several documentation issues were spotted: something must have gone wrong in a later rendering of the site (maybe in latest _{pkgdown}_ version or something)
1. In the reference page for [derive_params_growth_age()](https://github.com/pharmaverse/admiralpeds/blob/main/R/derive_params_growth_age.R) the formula is not rendering correctly
Image

Similarly for [derive_params_growth_height()](https://github.com/pharmaverse/admiralpeds/blob/main/R/derive_params_growth_height.R)
Image

2. For **each** WHO metadata entry on the [reference page](https://github.com/pharmaverse/admiralpeds/blob/main/R/data.R), the data type appears first, which may be confusing. If this information is necessary, consider presenting it differently to improve clarity.

Image

### PR DESCRIPTION
**Pull request by Julian-Guillo**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiralpeds (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiralpeds site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the Reference page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

File: R/data.R [hunk 1]
```diff
@@ -106,10 +106,10 @@
 #' day 1856
 #' @format A data frame with 1857 rows and 4 variables:
 #' \describe{
-#'   \item{\code{Day}}{integer Age in days}
-#'   \item{\code{L}}{double Box-Cox transformation for normality}
-#'   \item{\code{M}}{double Median}
-#'   \item{\code{S}}{double Coefficient of variation}
+#'   \item{\code{Day}}{Age in days (*integer*)}
```
Comment (Fanny-Gautier) [inline R/data.R:109]: @rossfarrugia Is the type really necessary, or am I the only one finding it confusing?
image

**Review by Fanny-Gautier — CHANGES_REQUESTED** [review]
@Julian-Guillo thank you for the updates. Please review and implement the below comments. Let me know if you have any questions. Thank you.

File: R/data.R (hunk 1 above)
Comment (Julian-Guillo) [reply R/data.R:109]: News file updated with requested changes. Will wait on your decision on this before adding more commits to the `R/data.R` file. Thanks for the feedback!

Comment (Fanny-Gautier) [reply R/data.R:109]: If data types are to be dropped, for both `WHO` and `CDC` metadata, `NEWS.md` will have to be updated again

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# admiralpeds 0.2.1
+
+## Documentation
+
+- Improved documentation on function argument data types and solved equation rendering issue on website (#104)
```
Comment (Fanny-Gautier) [inline NEWS.md:5]:
```suggestion
- Improved display of data types in the `WHO` metadata and added data types to the `CDC` metadata (#104)
<details>
<summary>Developer Notes</summary>

* Default math-rendering updated to `mathjax` (#104)

</details>
```

File: R/data.R (hunk 1 above)
Comment (Fanny-Gautier) [reply R/data.R:109]: Hi @rossfarrugia ,
Could you please share your thoughts on whether it's necessary to display the data type for the metadata? Thank you.

Comment (rossfarrugia) [reply R/data.R:109]: @Fanny-Gautier sorry i missed this question earlier. i think these could be safely dropped personally

**Comment by Fanny-Gautier** [comment]
> @Fanny-Gautier Got a couple of warnings from the linter. They are about using implicit return instead of explicit on the `R/bin_utils.R` file, which I haven't touched on any of my commits. I believe they are more readable with the returns as they are right now, because they are early exits of the function inside a conditional. Should they be changed or be left as they are?
> 
> image

Are these more like notes than actual warnings?
If so, please tick `Address or fix all lintr warnings and errors - lintr::lint_package()` and the remaining unticked boxes to complete the PR.
